biorxiv/polka_subset_experiment/03_rerun_analyses_with_added_subset.py

```diff
@@ -0,0 +1,602 @@
+from scipy.stats import linregress
+import seaborn as sns
+from sklearn.model_selection import GridSearchCV
+from sklearn.linear_model import LogisticRegressionCV
+from sklearn.preprocessing import StandardScaler
+from sklearn.tree import DecisionTreeClassifier, export_graphviz
+import spacy
+import tqdm
+
+from annorxiver_modules.corpora_comparison_helper import (
+    aggregate_word_counts,
+    calculate_confidence_intervals,
+    get_term_statistics,
+)
+
+sys.path.append(str(Path("../../../preprint_similarity_search/server").resolve()))
+from SAUCIE import SAUCIE, Loader  # noqa: E402
+
+mpl.rcParams["figure.dpi"] = 250
+
+# +
+# Set up porting from python to R
+# and R to python :mindblown:
+
+import rpy2.rinterface  # noqa: E402
+
+# %load_ext rpy2.ipython
+# -
+
+# # Corpora Comparison Preprint-Published Update
+
+# ## BioRxiv to Published Mapping
+
+mapped_doi_df = (
+    pd.read_csv("../journal_tracker/output/mapped_published_doi.tsv", sep="\t")
+    .query("published_doi.notnull()")
+    .query("pmcid.notnull()")
+    .groupby("preprint_doi")
+    .agg(
+        {
+            "author_type": "first",
+            "heading": "first",
+            "category": "first",
+            "document": "first",
+            "preprint_doi": "last",
+            "published_doi": "last",
+            "pmcid": "last",
+        }
+    )
+    .reset_index(drop=True)
+)
+mapped_doi_df.head()
+
+polka_et_al_mapped_df = pd.read_csv(
+    "output/polka_et_al_pmc_mapped_subset.tsv", sep="\t"
+)
+polka_et_al_mapped_df.head()
+
+spacy_nlp = spacy.load("en_core_web_sm")
+stop_word_list = list(spacy_nlp.Defaults.stop_words)
+
+# ## BioRxiv Token Counts
+
+polka_preprints = list(Path("output/biorxiv_word_counts").rglob("*tsv"))
+
+# +
+preprint_count = aggregate_word_counts(polka_preprints)
+
+preprint_count_df = (
+    pd.DataFrame.from_records(
+        [
+            {
+                "lemma": token[0],
+                "pos_tag": token[1],
+                "dep_tag": token[2],
+                "count": preprint_count[token],
+            }
+            for token in preprint_count
+        ]
+    )
+    .query(f"lemma not in {stop_word_list}")
+    .groupby("lemma")
+    .agg({"count": "sum"})
+    .reset_index()
+    .sort_values("count", ascending=False)
+)
+
+preprint_count_df.head()
+# -
+
+# ## PMCOA Token Counts
+
+polka_published = list(Path("output/pmcoa_word_counts").rglob("*tsv"))
+
+# +
+published_count = aggregate_word_counts(polka_published)
+
+published_count_df = (
+    pd.DataFrame.from_records(
+        [
+            {
+                "token": token[0],
+                "pos_tag": token[1],
+                "dep_tag": token[2],
+                "count": published_count[token],
+            }
+            for token in published_count
+        ]
+    )
+    .query(f"token not in {stop_word_list}")
+    .groupby("token")
+    .agg({"count": "sum"})
+    .reset_index()
+    .sort_values("count", ascending=False)
+)
+
+published_count_df.head()
```

Comment:
Is there a reason that two of the lemmas are blank here? Are they whitespace

![image](https://user-images.githubusercontent.com/15317110/113145097-049b3a00-91fc-11eb-865f-80aeab1e7e1f.png)

Reply:
>Is there a reason that two of the lemmas are blank here? Are they whitespace

Yup those are spaces/tabs.